app/assets/javascripts/discourse/models/user.js.es6

```diff
@@ -615,17 +615,13 @@ const User = RestModel.extend({
     }
   },
 
-  ignore() {
-    return ajax(`${userPath(this.get("username"))}/ignore.json`, {
+  updateNotificationLevel(level) {
+    return ajax(`${userPath(this.get("username"))}/notification_level.json`, {
       type: "PUT",
-      data: { ignored_user_id: this.get("id") }
-    });
-  },
-
-  unignore() {
-    return ajax(`${userPath(this.get("username"))}/ignore.json`, {
-      type: "DELETE",
-      data: { ignored_user_id: this.get("id") }
+      data: {
+        notification_level: level,
+        notification_level_user_id: this.get("id")
```

Comment:
Do we actually need that parameter since we already know what user we're muting via their username in the route?

Reply:
Good point. On the `UsersController`, it seems that we depend on `fetch_user_from_params`, I will do so.